login-Signup.css

```
@@ -0,0 +1,237 @@
+* {
+    box-sizing: border-box;
+}
+
+@media (min-width: 375px) {
+
+    /* 버튼 관련 */
+    .buttons {
+        width: 100%;
+        height: 56px;
+        border-radius: 40px;
+        background-color: #9CA3AF;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        color: #F3F4F6;
+        margin-bottom: 24px;
+        font-size: 20px;
+        font-weight: 600;
+        border: 0px;
+    }
+
+    /* 전체적인 틀 */
+    body {
+        font-family: "Pretendard Variable", Pretendard, sans-serif;
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    container {
+        margin-top: 80px;
+        width: 100%;
+        max-width: 400px;
+        height: auto;
+        margin-right: 16px;
+        margin-left: 16px;
+    }
+
+    /* 로고관련 */
+    #logo > a {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        gap: 11px;
+        margin-bottom: 24px;
+    }
+
+    #logo img:first-child {
+        width: 52px;
+    }
+
+    #logo img:last-child {
+        width: 133px;
+    }
+
+    /* 폼테그 관련 */
+    form {
+        display: flex;
+        flex-direction: column;
+        color: #1F2937;
+        font-size: 18px;
+        font-weight: 700;
+    }
+
+    form > label {
+        margin-bottom: 8px;
+    }
+
+    form > input {
+        width: 100%;
+        height: 56px;
+        border: 0;
+        background-color: #F3F4F6;
+        border-radius: 12px;
+        margin-bottom: 16px;
+    }
+
+    input::placeholder {
+        font-size: 16px;
+        padding: 24px;
+    }
+
+    /* sns 로그인 관련 */
+    .sns-login {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        background-color: #E6F2FF;
+        padding: 16px 23px;
+        border-radius: 8px;
+        margin-bottom: 24px;
+        font-size: 16px;
+        font-weight: 500;
+    }
+
+    .sns-login-image {
+        display: flex;
+        flex-direction: row;
+        gap: 16px;
+    }
+
+    .sns-login-image img {
+        width: 42px;
+        height: 42px;
+    }
+
+    /* footer 관련 */
+    .footer {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: center;
+        gap: 4px;
+    }
+
+    .footer p {
+        margin: 0px;
+    }
+
+    .footer a {
+        color: #3692FF;
+    }
+}
+ 
+@media (min-width: 768px) {
+/* 버튼 관련 */
+    .buttons {
+        width: 100%;
+        height: 56px;
+        border-radius: 40px;
+        background-color: #9CA3AF;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        color: #F3F4F6;
+        margin-bottom: 24px;
+        font-size: 20px;
+        font-weight: 600;
+        border: 0px;
+    }
```

❗️ 수정요청
해당 미디어 쿼리 조건을 보면 `@media (min-width: 375px)` 로 375px 이상의 화면에서는 해당 css가 적용되게 됩니다. 지금 작성하신 미디어 쿼리 조건인 `@media (min-width: 768px)`를 만족하면서 그 전 조건도 만족할 수 있기 때문에 중복으로 스타일링을 해주실 필요가 없습니다.

변경이 필요한 경우만 작성해주시고 중복되는 것은 지워주시는 것이 성능, 유지보수 및 가독성 측면에서 유리합니다~

```suggestion
@media (min-width: 768px) {
```